packages/components/scripts/vendorism/custom/tab.vendorism.js

```diff
@@ -0,0 +1,118 @@
+import {
+  addSectionsAfter, addSectionsBefore, replaceSections,
+} from '../replace-section.js';
+
+const FILES_TO_TRANSFORM = [
+  'tab.component.ts',
+  'tab.test.ts',
+];
+
+/**
+ * Transform the component code
+ * @param {String} path
+ * @param {String} originalContent
+ * @returns
+ */
+const transformComponent = (path, originalContent) => {
+  let content = addSectionsAfter([
+    // Add prefix slot documentation
+    [
+      '* @slot - The tab\'s label.',
+      ' * @slot prefix - Used to prepend an icon or similar element to the tab.',
+    ],
```

Comment:
Can we try to bring that to Shoelace? I'm realizing here, that we introduce this here – in fact this could be easily done inside the main slot + CSS as well, or what do you think?

Reply:
Yah, please talk with Lea. I think we can easily just show it in a template.

Reply:
Done